instances/treasury-devdao.near/widget/pages/dashboard/index.jsx

```diff
@@ -1 +1,211 @@
-return <h1>Dashboard</h1>;
+const treasuryAccount = "${REPL_TREASURY}";
+
+const Wrapper = styled.div`
+  min-height: 80vh;
+  .flex-1 {
+    flex: 1;
+  }
+
+  .text-sm {
+    font-size: 12px;
+  }
+
+  .border-bottom {
+    border-bottom: 1px solid var(--border-color);
+  }
+
+  .text-align-end {
+    text-align: end !important;
+  }
+
+  .pointer {
+    cursor: pointer;
+  }
+
+  .dashboard-item > div {
+    min-width: 400px;
+  }
+`;
+
+const [nearStakedTokens, setNearStakedTokens] = useState(null);
+
+const balanceResp = fetch(
+  `https://api3.nearblocks.io/v1/account/${treasuryAccount}`
+);
+const nearBalance = Big(balanceResp?.body?.account?.[0]?.amount ?? "0")
+  .div(Big(10).pow(24))
+  .toFixed();
+
+const nearPrice = useCache(
+  () =>
+    asyncFetch(`https://api3.nearblocks.io/v1/charts/latest`).then((res) => {
+      return res.body.charts?.[0].near_price;
+    }),
+  "price",
+  { subscribe: false }
+);
+
+const userFTTokens = useCache(
+  () =>
+    asyncFetch(
+      `https://api3.nearblocks.io/v1/account/${treasuryAccount}/inventory`
+    ).then((res) => {
+      const fts = res.body.inventory.fts;
+      const amounts = fts.map((ft) => {
+        const amount = ft.amount;
+        const decimals = ft.ft_meta.decimals;
+        const tokensNumber = Big(amount ?? "0")
+          .div(Big(10).pow(decimals))
+          .toFixed();
+        const tokenPrice = ft.ft_meta.price;
+        return Big(tokensNumber)
+          .mul(tokenPrice ?? 0)
+          .toFixed();
+      });
+      return {
+        totalCummulativeAmt: amounts.reduce(
+          (acc, value) => acc + parseFloat(value),
+          0
+        ),
+        fts,
+      };
+    }),
+  "all-token-amount",
+  { subscribe: false }
+);
+
+const loading = (
+  <Widget src={"${REPL_DEVHUB}/widget/devhub.components.molecule.Spinner"} />
+);
+
+const code = `
+  <!doctype html>
```

Comment:
Also in this iframe you don't show anything, so only the script tag should be needed.